website/frontend/management/commands/scraper.py

```diff
@@ -6,6 +6,7 @@
 import httplib
 import logging
 import os
+import smtplib
```

Comment:
You're not using this?

Reply:
Correct.  I was going to add checks for this sort of stuff in the build.